wallet/src/main/java/io/neow3j/wallet/Wallet.java

```diff
@@ -166,35 +215,35 @@ public NEP6Wallet toNEP6Wallet() {
         return new NEP6Wallet(name, version, scryptParams, accts, null);
     }
 
-    public static Builder fromNEP6Wallet(String nep6WalletFileName) throws IOException {
+    public static Wallet fromNEP6Wallet(String nep6WalletFileName) throws IOException {
         return fromNEP6Wallet(
                 Wallet.class.getClassLoader().getResourceAsStream(nep6WalletFileName));
     }
 
-    public static Builder fromNEP6Wallet(URI nep6WalletFileUri) throws IOException {
+    public static Wallet fromNEP6Wallet(URI nep6WalletFileUri) throws IOException {
         return fromNEP6Wallet(nep6WalletFileUri.toURL().openStream());
     }
 
-    public static Builder fromNEP6Wallet(File nep6WalletFile) throws IOException {
+    public static Wallet fromNEP6Wallet(File nep6WalletFile) throws IOException {
         return fromNEP6Wallet(new FileInputStream(nep6WalletFile));
     }
 
-    public static Builder fromNEP6Wallet(InputStream nep6WalletFileInputStream) throws IOException {
+    public static Wallet fromNEP6Wallet(InputStream nep6WalletFileInputStream) throws IOException {
         NEP6Wallet nep6Wallet = OBJECT_MAPPER
                 .readValue(nep6WalletFileInputStream, NEP6Wallet.class);
         return fromNEP6Wallet(nep6Wallet);
     }
 
-    public static Builder fromNEP6Wallet(NEP6Wallet nep6Wallet) {
+    public static Wallet fromNEP6Wallet(NEP6Wallet nep6Wallet) {
```

Comment:
A NEP6Wallet will contain a account marked as default. I.e. you need to look for the default account in the NEP6Wallet and set it on the Wallet.